### PR TITLE
Basic link forwarding works

### PR DESCRIPTION
**Pull request by cwRichardKim**
I would want to look into whether I can change the way the files are named.  Other than that, this works!

### CODE REVIEW

File: _layouts/links.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<head><meta http-equiv="refresh" content="0; URL='http://{{ page.forward-to }}'" /></head>
```
Comment (tekknolagi) [inline _layouts/links.html]: I think preferred style is `snake_case`.

File: index.html [hunk 2]
```diff
@@ -9,15 +9,11 @@ <h1 class="page-heading">Posts</h1>
   <ul class="post-list">
     {% for post in site.posts %}
```
Comment (tekknolagi) [inline index.html:10]: Do we want to list all links?

**Review by tekknolagi — APPROVED** [review]
Can we also add a button that links to the open a PR page so that people can add links?

File: _layouts/links.html (hunk 1 above)
Comment (cwRichardKim) [reply _layouts/links.html]: Are you sure?  The file system defaults to using dashes
edit: nope nvm you're right, just looked it up

File: index.html (hunk 2 above)
Comment (cwRichardKim) [reply index.html:10]: working on the frontpage next.  This is just step 1

**Comment by tekknolagi** [comment]
LGTM.

**Pull request merged**